packages/components/src/ListBox/tests/chromatic/ListBox.stories.tsx

```diff
@@ -110,47 +110,87 @@ export const Default = {
                 <h1>Items with Start Icon</h1>
                 <Inline alignY="flex-start">
                     <ListBox {...args} size="lg">
-                        <ListBoxItem id="1" textValue="Earth"><SparklesIcon /><Text slot="label">Earth</Text></ListBoxItem>
+                        <ListBoxItem id="1" textValue="Earth"><SparklesIcon /><Text
+                            slot="label"
+                        >Earth</Text></ListBoxItem>
```

Comment:
I don't like the way it's indented. If you don't wan tit on one line, I'd put it like this:
```suggestion
                        <ListBoxItem id="1" textValue="Earth">
                            <SparklesIcon />
                            <Text slot="label">Earth</Text>
                        </ListBoxItem>
```

This goes for all the examples here.

Reply:
I didn't choose the linter because I didn't like the formatting.